components/automate-ui/src/app/modules/infra-proxy/cookbook-details/cookbook-details.component.html

```diff
@@ -0,0 +1,30 @@
+<div class="content-container">
+  <div class="container">
+    <main>
+      <chef-breadcrumbs>
+        <chef-breadcrumb [link]="['/infrastructure/chef-servers']">Chef Server</chef-breadcrumb>
+        <chef-breadcrumb [routerLink]="['/infrastructure/chef-servers', serverId]">Orgs</chef-breadcrumb>
+        <chef-breadcrumb [routerLink]="['/infrastructure/chef-servers', serverId, 'org', orgId, 'cookbooks']">Cookbooks
+        </chef-breadcrumb>
+        {{ cookbookName }}
+      </chef-breadcrumbs>
+      <chef-page-header>
+        <chef-heading>{{ cookbookName }}</chef-heading>
+        <div class="version-dropdown">
+          <label>Version</label>
+          <chef-select *ngIf="cookbook"
+            (change)="handleCookbookVersionChange(serverId, orgId, cookbookName, $event)">
+            <chef-option *ngFor="let version of cookbook.versions" [value]="version">{{ version }}</chef-option>
+          </chef-select>
+        </div>
+        <chef-tab-selector [value]="tabValue">
+          <chef-option value='details' data-cy="content-tab">Details</chef-option>
+        </chef-tab-selector>
+      </chef-page-header>
+      <section class="page-body">
+        <chef-loading-spinner *ngIf="cookbookDetailsLoading" size="50"></chef-loading-spinner>
+        <chef-markdown *ngIf="readFileContent" [text]="readFileContent.content"></chef-markdown>
```

Comment:
I am using stencil `chef-markdown` atom to show cookbook version details `README` file content getting some console errors, happy to have a discussion on the same.

Reply:
This issue (#3462 ) has been resolved by @scottopherson (#3473). Thank you @scottopherson :+1: